test/SCBridgeWallet.test.ts

```diff
@@ -80,14 +81,21 @@ describe("SCBridgeWallet", function () {
 
     const n = await ethers.provider.getNetwork();
 
+    const deployer = await hre.ethers.getContractFactory("SCBridgeWallet");
+
+    const payeeSCW = await deployer.deploy(
+      ethers.Wallet.createRandom(),
+      intermediary,
+      entrypoint,
+    );
+
     // Generate a random payee address that we can use for the transfer.
-    const payee = hre.ethers.Wallet.createRandom().address;
 
     // Encode calldata that calls the execute function to perform a simple transfer of ether to the payee.
     const callData = nitroSCW.interface.encodeFunctionData("execute", [
-      payee,
+      await payeeSCW.getAddress(),
       ethers.parseEther("0.5"),
-      ZeroHash,
+      "0x",
```

Comment:
When updating this test to use the SCW as a payee, using the `ZeroHash` as the `func` data meant the transfer didn't happen. I think this was because it was trying to interpet the non-zero `func` data as a contract call. 

By switching to empty `func` data the `execute` function successfully transferred the funds